src/clients/app-client.js

```diff
@@ -1,12 +1,15 @@
 'use strict'
 
-const Client = require('./base-browser-client')
+const Client = require('./base-client')
 
 class AppClient {
-  constructor (url) {
+  constructor (url, options = {}) {
     this.client = new Client({
       name: 'AppsClient',
       url: url,
+      accessKey: options.accessKey,
+      secretKey: options.secretKey,
+      doorkeeperBaseUrl: options.doorkeeperBaseUrl,
       contentType: 'application/vnd.api+json'
```

Comment:
`contentType: 'application/vnd.api+json'` can be removed in every client class as it is default

Reply:
I know, but sometimes I want to be explicit. 😄 
If someone looks into this client implementation, he would see at a first glance that it's a JSONAPI client.

I can still remove it though.